specification/consumption/resource-manager/Microsoft.Consumption/stable/2022-06-01/consumption.pricesheets.operations.json

```diff
@@ -0,0 +1,511 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-06-01",
+    "title": "ConsumptionManagementClient",
+    "description": "Consumption management client provides access to consumption resources for Azure Enterprise Subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Consumption/pricesheets/default": {
+      "get": {
+        "tags": [
+          "PriceSheet_20220601"
+        ],
+        "operationId": "PriceSheet_Get20220601",
```

Comment:
why all the operationIds  have 20220601 suffix ? Is it a little weird to have number suffix for operations ?

Reply:
@jayati-tehri , can you reply?

Reply:
Agree we should not allow that. It just result to bad SDK.

Reply:
I added the suffix to this operation id because it was throwing me an error. that this is already being used in consumption.json file version 2021-10-01. My swagger python pipeline was failing with duplicate schema errors. Hence I renamed this to a different name. Since operation id is just unique name assigned I thought to it's best to add version id to the unique name.

Reply:
@weidongxu-microsoft please request changes.
@mehulviby I never know operationId could be resolved in such a creative way. But we are actually asking to give the operationId a more meanful name, because we will use operationId to generate the SDK. If we have operationId like PriceSheet_Get20220601, we will get a SDK user experience like `client.priceSheet.get20220601()` .

Reply:
@mehulviby 

First please check whether your swagger will co-exist with consumption.json.

If not, you probably should break them to 2 different tags.

If yes (that is your current readme), you either use a different name e.g. `PriceSheet_GetPremium`, or different group e.g. `PriceSheetPremium_Get`